src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Actions/Utils/SqlHelper.cs

```diff
@@ -37,11 +37,48 @@ internal static string GetSolutionAboutLink(string solutionId, string connection
             return result;
         }
 
+        public static int GetSolutionStatus(string solutionId, string connectionString)
+        {
+            var query = "SELECT SupplierStatusId from [dbo].[Solution] where Id=@solutionId";
+            
+            SqlParameter[] parameters = new SqlParameter[] {
+                new SqlParameter("@solutionId", solutionId)
+            };
+
+            var result = SqlReader.Read<int>(connectionString, query, parameters, GetSupplierStatus);
+
+            return result;
+        }
+
+        private static int GetSupplierStatus(IDataReader dr)
+        {
+            dr.Read();
+            int.TryParse(dr["SupplierStatusId"].ToString(), out int result);
+            return result;
+        }
+
+        public static void ResetSolutionStatus(string solutionId, string connectionString)
+        {
+            var query = $"UPDATE Solution set SupplierStatusId=1 where Id=@solutionId";
+
+            SqlParameter[] parameters = new SqlParameter[] {
+                new SqlParameter("@solutionId", solutionId)
+            };
+
+            SqlReader.Read(connectionString, query, parameters, NullReturn);
+        }
```

Comment:
Could be named differently (name doesn't reflect what user would expect)
Suggestion: ResetSolutionStatusForSolutionId | do a reset to 1 no matter the id